boa3/internal/analyser/astanalyser.py

```diff
@@ -80,6 +80,8 @@ def visit(self, node: ast.AST) -> Any:
             self._log_error(error)
         except CompilerWarning as warning:
             self._log_warning(warning)
+        except KeyboardInterrupt:
+            raise KeyboardInterrupt() from None
```

Comment:
is it needed to instantiate another error? 
doesn't the following work?
```suggestion
        except KeyboardInterrupt as e:
            raise e
```